packages/tools/locales/fr_FR.po

```diff
@@ -1058,7 +1058,7 @@ msgstr ""
 
 #: packages/lib/Synchronizer.ts:192
 msgid "Deleted local items: %d."
-msgstr "Objets supprimés localement : %d."
+msgstr "Objets local supprimés : %d."
```

Comment:
While we're at it, let's make it even shorter by changing "supprimés" to just "suppr."

Reply:
ok for me but I am not sure that introducing abbreviation in the app is a good idea in general?).